### PR TITLE
Sopp and tle fetcher edits

### PR DESCRIPTION
**Pull request by ellem95**

### CODE REVIEW

**Review by nikojpapa — CHANGES_REQUESTED** [review]

File: utilities.py
[the remark this answers is not in the document]
Comment (ellem95) [reply utilities.py:1]: Moved

File: utilities.py [hunk 1]
```diff
@@ -37,3 +37,6 @@ def convert_datetime_to_utc(localtime: datetime) -> datetime:
 
 def get_script_directory(module) -> Path:
     return Path(os.path.dirname(os.path.realpath(module)))
+
+def get_root_directory(module) -> Path:
+    return Path(os.path.dirname(os.path.abspath(__file__)))
```
[the remark this answers is not in the document]
Comment (ellem95) [reply utilities.py]: Made this change and use `get_supplements_directory` in the TleFetcher now.

File: utilities.py [hunk 2]
```diff
@@ -37,3 +37,6 @@ def convert_datetime_to_utc(localtime: datetime) -> datetime:
 
 def get_script_directory(module) -> Path:
     return Path(os.path.dirname(os.path.realpath(module)))
+
+def get_root_directory(module) -> Path:
```
[the remark this answers is not in the document]
Comment (ellem95) [reply utilities.py]: Changed

[the remark this answers is not in the document]
Comment (ellem95) [reply utilities.py]: Changed

**Review by nikojpapa — APPROVED** [review]

**Pull request merged**